src/app.css

```diff
@@ -0,0 +1,22 @@
+html {
+  box-sizing: border-box;
+}
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+body {
+  margin: 0;
+}
+
+@import "~@variant/profile/colors/index.module.css";
+@import "~@variant/profile/typography/index.module.css";
+
+#root {
+  position: relative;
+}
+
+html {
+  background: var(--color-secondary4__tint2);
```

Comment:
Any reason to separate this rule from the other rule at the top of the file?

Reply:
Not really, but just separating the "global general configuration" from the project specific. And in order for some specificity from the profile/color included above.

Reply:
I see 👌